src/test/java/au/gov/nla/dlir/util/MarcDataHelperTest.java

```diff
@@ -0,0 +1,153 @@
+package au.gov.nla.dlir.util;
+
+import au.gov.nla.dlir.models.Content;
+import au.gov.nla.dlir.models.bibdata.MarcData;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.Sets;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class MarcDataHelperTest {
+    private static final String CATALOGUE_URL = "http://catalogue.nla.gov.au/Record/{bibId}?view=json_record";
+    private static final String COLLECTION_SERVICE_URL = "http://dl-devel.nla.gov.au/nla-cat/record/{bibId}?format=json&includeSuppressed=false";
+
+    private static final List<String> bibIds = new ArrayList<>();
+
+    @Before
+    public void setup() throws IOException {
+        InputStream is = ClassLoader.getSystemResourceAsStream("bibid/bib1001-2000.txt");
+        BufferedReader br = new BufferedReader(new InputStreamReader(is));
+        String line;
+        while ((line = br.readLine()) != null){
+            bibIds.add(line);
+        }
+        br.close();
+    }
+
+    @Test
+    @Ignore
+    public void test(){
```

Comment:
@terenceingram , This is the integration test to compare the bib data returned from the legacy catalogue service with the new bib data converted from the marc data returned from the new collection service. 

This trial and error approach is taken because no one knows the exact rule of mapping. The more bib ids are tested and errors fixed, the more accurate the set of rules will be.